src/distributionColumns.psql

```diff
@@ -9,4 +9,5 @@ SELECT create_distributed_table('resources', 'organization_id');
 SELECT create_distributed_table('resource_creator_mapping', 'creator_id');
 SELECT create_distributed_table('review_resources', 'reviewer_id');
 SELECT create_distributed_table('review_stages', 'organization_id');
-SELECT create_distributed_table('reviews', 'organization_id');
\ No newline at end of file
+SELECT create_distributed_table('reviews', 'organization_id');
+SELECT create_distributed_table('activities', 'organization_id');
```

Comment:
@priyanka-TL Please add resource id as DC and make corresponding changes

Reply:
yes done